drivers/Makefile.dep

```diff
@@ -20,6 +20,13 @@ ifneq (,$(filter at86rf2%,$(USEMODULE)))
   USEMODULE += netif
   USEMODULE += ieee802154
   USEMODULE += netdev_ieee802154
+  ifeq (,$(filter gnrc_netif2,$(USEMODULE)))
+    ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
+      # XXX: this can be modelled as a dependency for gnrc_netdev_default as soon
```

Comment:
This looks like what I removed with: https://github.com/RIOT-OS/RIOT/pull/7677 so I think it can be dropped.